wolfcrypt/src/port/cavium/README_Octeon.md

```diff
@@ -0,0 +1,150 @@
+# Octeon III
```

Comment:
I'm not sure if we can/should make these instructions public. Also please replace the "dgarske" references with `"<username>"`.

Reply:
I replaced this file with an Octeon version of the other README.